CloudFabric.EventSourcing.AspNet/CloudFabric.EventSourcing.AspNet.Postgresql/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -12,10 +13,11 @@ public static class ServiceCollectionExtensions
         public static IEventSourcingBuilder AddPostgresqlEventStore(
             this IServiceCollection services,
             string eventsConnectionString,
-            string tableName
+            string eventTableName,
+            string itemTableName
         )
         {
-            var eventStore = new PostgresqlEventStore(eventsConnectionString, tableName);
+            var eventStore = new PostgresqlEventStore(eventsConnectionString, eventTableName, itemTableName);
```

Comment:
eventsTableName, itemsTableName for consistency with cosmosdb and other providers